x/interchainquery/genesis_test.go

```diff
@@ -0,0 +1,89 @@
+package interchainquery_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	ibctesting "github.com/cosmos/ibc-go/v3/testing"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/persistenceOne/persistence-sdk/simapp"
+	"github.com/persistenceOne/persistence-sdk/x/interchainquery"
+	"github.com/persistenceOne/persistence-sdk/x/interchainquery/keeper"
+	"github.com/persistenceOne/persistence-sdk/x/interchainquery/types"
+)
+
+func init() {
+	ibctesting.DefaultTestingAppInit = simapp.SetupTestingApp
+}
+
+func TestInterChainQueryTestSuite(t *testing.T) {
+	suite.Run(t, new(InterChainQueryTestSuite))
+}
+
+type InterChainQueryTestSuite struct {
+	suite.Suite
+
+	coordinator *ibctesting.Coordinator
+
+	chainA *ibctesting.TestChain
+	chainB *ibctesting.TestChain
+	path   *ibctesting.Path
+}
+
+func (s *InterChainQueryTestSuite) GetSimApp(chain *ibctesting.TestChain) *simapp.SimApp {
+	app, ok := chain.App.(*simapp.SimApp)
+	if !ok {
+		panic("not sim app")
+	}
+
+	return app
+}
+
+func (s *InterChainQueryTestSuite) SetupTest() {
+	s.coordinator = ibctesting.NewCoordinator(s.T(), 2)
+	s.chainA = s.coordinator.GetChain(ibctesting.GetChainID(1))
+	s.chainB = s.coordinator.GetChain(ibctesting.GetChainID(2))
+
+	s.path = newSimAppPath(s.chainA, s.chainB)
+	s.coordinator.SetupConnections(s.path)
+}
+
+func newSimAppPath(chainA, chainB *ibctesting.TestChain) *ibctesting.Path {
```

Comment:
Move this function either above or bellow all the other function definitions. Ideally just move it above all functions.